src/App.js

```diff
@@ -12,42 +12,112 @@ import React from "react";
 // components
 //=============================
 
-//=============================
-// containers
-//=============================
 
-//=============================
-// hooks
-//=============================
+export class AddTodo extends React.Component {
 
-//=============================
-// styles
-//=============================
+    constructor() {
+        super();
+        this.state = {
+            todo: '',
+        }
+    }
+
+    updateInput = (e) => {
+        this.setState({todo: e.target.value});
+    };
+
+    submitTodo = (e) => {
+        e.preventDefault();
+        console.log(this.state);
+    };
```

Comment:
The updated `methods` allow me to prevent the default `form behavior` and log the current state upon submission